README.md

```diff
@@ -0,0 +1,235 @@
+# Katzenpost Mix Network
+
+## Post Quantum Anonymous Communication Network
+
+![build badge](https://github.com/katzenpost/katzenpost/actions/workflows/go.yml/badge.svg?branch=main)
+
+Katzenpost is a software project dedicated to designing and
+implementing mix network protocols. A mix network is a type of
+anonymous communication network. An anonymous communication
+network is also known as a traffic analysis resistant network; that is,
+it's protocols are designed to resist statistical analysis by passive
+global adversaries.
+
+Traffic analysis typically refers to the statistical analysis of
+encrypted traffic. Traffic analysis is worth defending against given
+that common network protocols leak lots of information such as source
+and destination IP addresses, message size, message sequence, message
+delay pattern, geographical locations, social graph etc. Mere end to
+end encryption alone cannot protect against this type of information
+leakage.
+
+At the most basic level, mixnets are composed of mix nodes. These are
+a cryptographic packet switching routers which protect our privacy by
+mixing many messages together and cryptographically transforming them
+before routing them on to the next hop. Mix nodes also use shuffling
+or added latency to create uncertainty for network observers. This
+uncertainty is in regards to trying to link incoming messages with the
+outgoing messages.
+
+
+# Project Status
+
+Many excited new changes are forthcoming!
+You can watch our progress being tracked, here:
+
+https://github.com/orgs/katzenpost/projects/6/views/5
+
+
+There are essentially two types of interaction with a Katzenpost mixnet:
+1. clients talk to mixnet services and their traffic stays in the mixnet
+2. clients talk to Internet services; proxying through the mixnet onto the Internet.
+
+# Client:
+
+Currently, [Katzen](https://github.com/katzenpost/katzen) is the only client available for use with Katzenpost. However a SOCKS proxy client is forthcoming
+and you'll be able to use that with many existing applications.
+
+# Server Side: Easy One Step BUILD
+
+You'd want to build the server side if you are
+a developer, mix node operator or dirauth operator.
+
+This builds 3 binaries:
+
+* mix server
+* dirauth server
+* mixnet ping CLI command
+
+All you have to do is type:
+
+```bash
+make
```

Comment:
note that you'll need to have the build deps installed, perhaps refer to the more detailed instructions inthe docker path?

Reply:
https://github.com/katzenpost/katzenpost/issues/358